sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/SendSmsToGroupExample.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.sms.samples.quickstart;
+
+import com.azure.communication.sms.SmsClient;
+import com.azure.communication.sms.SmsClientBuilder;
+import com.azure.communication.sms.models.SmsSendOptions;
+import com.azure.communication.sms.models.SmsSendResult;
+import com.azure.core.util.Context;
+import java.util.Arrays;
+
+public class SendSmsToGroupExample {
+    public static void main(String[] args) {
+        String connectionString = System.getenv("COMMUNICATION_CONNECTION_STRING");
```

Comment:
This will need to be done separately from this PR, but we're going to eventually change this env variable once we switch to using static resource. Just FYI